BricklinkSharp.Client/BricklinkClient.cs

```diff
@@ -794,4 +838,14 @@ public void Dispose()
 
         return result;
     }
+
+
+    private async void _measureRequest(CancellationToken cancellationToken = default)
```

Comment:
Naming convention -> MeasureRequestAsync + async methods should return a task:
```suggestion
    private async Task MeasureRequestAsync(CancellationToken cancellationToken = default)
```